kvmagent/ansible/kvm.py

```
@@ -273,15 +282,18 @@ def rpm_based_install():
-            	host_post_info.post_label = "ansible.shell.install.pkg"
-            	host_post_info.post_label_param = "libvirt-python"
-            	(status, output) = run_remote_command(command, host_post_info, True, True)
-            	if status is True:
-                    dep_list = ' '.join([pkg for pkg in dep_list.split() if not pkg.startswith("libvirt")])
+                # python3-libvirt installation does not affect the libvirt installation
+                command = (
+                    "yum --disablerepo=* --enablerepo={0} --assumeno install python3-libvirt |awk '{{print $1}}' | grep -Ew '^\s*libvirt\s*$'").format(
+                    zstack_repo)
+                host_post_info.post_label = "ansible.shell.install.pkg"
+                host_post_info.post_label_param = "python3-libvirt"
+                (status, output) = run_remote_command(command, host_post_info, True, True)
```

Ruff (0.12.2)

`run_remote_command` may be undefined, or defined from star imports

(F405)